### DISCUSSION TITLE
Failure to deploy

### DISCUSSION THREAD

**Discussion by dianaspetcu** [illacloud/illa-builder#2476] [English Support]
Hi team,

I have an issue with the deploy button. Every time when I try to press the button deploy an error appears. (I also have a premium account if it matter) And the only thing that works is to insert manually  " http://......./deploy "at the and of html link from the bar. Can you fix this bug?

Thanks,
Diana

![MicrosoftTeams-image (1)](https://github.com/illacloud/illa-builder/assets/132670969/4235d4b3-1954-4086-8d9d-dd7e53dd7391)

**Comment by donqu1xotechina** [comment]
Hey Diana:
We will let our engineer check this out ASAP. Meanwhile, We would suggest you join our Discord Community for more support.

best

ILLA Team
